src/index.ts

```diff
@@ -342,9 +342,17 @@ export async function main(
     UploadLogger.verbose(`${parameter}`)
   }
 
-  const validSlug =  checkSlug(buildParams.slug)
-  if (!validSlug) {
-    throw new Error(`Slug must follow the format of "<owner>/<repo>" or be blank. We detected "${buildParams.slug}"`)
+  if (!hasToken) {
+    if (!buildParams.slug) {
+      throw new Error(`Slug must be set if a token is not passed`)
```

Comment:
Maybe add a line on how to set the slug (or if it's an env variable)

Reply:
👍